src/javascript/ContentEditor/SelectorTypes/Picker/Picker.jsx

```diff
@@ -123,6 +125,14 @@ export const Picker = ({
         fieldData, error, loading, notFound
     } = usePickerInputData(value && toArray(value));
 
+    const dispatch = useDispatch();
+
+    useEffect(() => {
```

Comment:
discussed with @r3dm1ke and this should only get triggered the first time loading as defaultViewMode shouldn't change after that.